charts/digger-backend/values.yaml

```diff
@@ -88,7 +88,7 @@ digger:
     # if the DB connection string is already in a secret
     # DB connection string should be in a key names DATABASE_URL
     # compatible with https://github.com/kloeckner-i/db-operator, for example
-    existingSecretName: "new-pg-creds"
+    existingSecretName: ""
     existingSecretKey: "postgres-password"
```

Comment:
I think the existingSecretKey and the example in the comment should probably match.  Obviously I'm partial to `DATABASE_URL` since that was my example, but consistency is more important. :)  Also, I guess I misspelled "named" in that comment. :/

Reply:
Actually, this implementation is still somewhat incompatible with the DB Operator mechanism.

When using [db-operator](https://github.com/db-operator/db-operator) to create the database, a secret is created which contains the entire connection URL.  One can set up a Secret Template with multiple values, as [in the docs](https://github.com/db-operator/db-operator/blob/main/docs/creatingdatabases.md#creatingdatabases), but typically it's preferable to just let the operator or other secret manager (I've used AWS systems manager secret driver and RDS for this before) build the whole connection string and inject that into the namespace.

So the outcome of the prior change is that I'd have to use something like kustomize to patch a secret with some other environment variables post-render.  The change introduced in #7 and #8 provides a way for DATABASE_URL to be brought in directly from an external secret. Basically the postgres.existingSecretName was hijacked in #10 to change from actually containing the full URL secret to setting an entirely different environment variable used later to build the connection string.  What ideally should've happened was to create two separate configuration mechanisms: one for specifying the connection string components, and a separate one for specifying the complete string.

Reply:
So I improved the comment, regarding the DB operator, my basic understanding is that we currently accept individual DB URL string components and form it ourselves. Perhaps to support accepting entire string as a secret we can try to override the value [here](https://github.com/diggerhq/helm-charts/blob/main/charts/digger-backend/templates/backend-deployment.yaml#L52) and accept it as a secret rather than an env variable

But I think it i something for another PR!

Reply:
Accepting the whole URL worked before #10. ;)

I'll submit support again in a new PR since I strongly suspect I'm the only one who currently uses it.